test/TestExtensions/API.Test/VSSolutionHelper.cs

```diff
@@ -467,7 +467,7 @@ public static async Task<EnvDTE.Project> GetProjectAsync(EnvDTE80.Solution2 solu
             return solution2
                 .Projects
                 .Cast<EnvDTE.Project>()
-                .FirstOrDefault(project => project.Name.Equals(projectName, StringComparison.OrdinalIgnoreCase));
+                .FirstOrDefault(project => { ThreadHelper.ThrowIfNotOnUIThread(); return project.Name.Equals(projectName, StringComparison.OrdinalIgnoreCase); });
```

Comment:
In the past when I tried to add a single line `if (val == null) throw new ArgumentNullException();`, I was told to add braces and new lines. So, if we still have that rule, I think this lamba method needs to be on multiple lines.

Reply:
Fixed.